kubernetes/beta/hmda-platform/values.yaml

```diff
@@ -26,11 +26,11 @@ hmda:
   serviceNamespace: beta.svc.cluster.local
 
 kafka:
-  institutionsTopic: institution
-  modifiedLarTopic: hmda-modified-lar
-  irsTopic: hmda-irs
-  analyticsTopic: hmda-analytics
-  signTopic: hmda-sign
+  institutionsTopic: beta-institution
+  modifiedLarTopic: beta-hmda-modified-lar
+  irsTopic: beta-hmda-irs
+  analyticsTopic: beta-hmda-analytics
+  signTopic: beta-hmda-sign
   disclosureTopic: hmda-spark-disclosure
```

Comment:
@PatrickHSI disclosure will not be deployed on beta. To not cause confusion, could we rename the topic to beta or remove it from here? I'd vote for the latter.